### PR TITLE
11.0 mig sale order lot selection

### PR DESCRIPTION
**Pull request by OpenCode**

### CODE REVIEW

**Review by ilmir-k — APPROVED** [review]
:+1:

File: sale_order_lot_selection/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# © 2015 Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    'name': 'Sale Order Lot Selection',
+    'version': '11.0.1.0.0',
+    'category': 'Sales Management',
+    'author': "Odoo Community Association (OCA), Agile Business Group",
+    'website': 'http://www.agilebg.com',
```
Comment (MeetKD) [inline sale_order_lot_selection/__manifest__.py:11]: website should be https://github.com/OCA/sale-workflow

File: sale_order_lot_selection/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2015 Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    'name': 'Sale Order Lot Selection',
+    'version': '11.0.1.0.0',
+    'category': 'Sales Management',
+    'author': "Odoo Community Association (OCA), Agile Business Group",
```
Comment (MeetKD) [inline sale_order_lot_selection/__manifest__.py:10]: Mention agile before OCA.

File: sale_order_lot_selection/tests/test_sale_order_lot_selection.py [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+        self.assertEqual(lot10_qty_available, 0)
+        lot11_qty_available = self._stock_quantity(
+            self.product_46, self.lot11, self.stock_location)
+        self.assertEqual(lot11_qty_available, 1)
+        lot12_qty_available = self._stock_quantity(
+            self.product_12, self.lot12, self.stock_location)
+        self.assertEqual(lot12_qty_available, 0)
+        # I'll try to confirm it to check lot reservation:
+        # lot11 has 1 availability and order4 has quantity 2
+        with self.assertRaises(Warning):
```
Comment (MeetKD) [inline sale_order_lot_selection/tests/test_sale_order_lot_selection.py:282]: You can remove the print stats and commented code if its not usable.

File: sale_order_lot_selection/model/__init__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+# GNU Affero General Public Licensefor more details.                    #
+#                                                                       #
+# You should have received a copy of the                                #
+# GNU Affero General Public License                                     #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>. #
+#                                                                       #
+#########################################################################
+
```
Comment (MeetKD) [inline sale_order_lot_selection/model/__init__.py:21]: I guess encoding part is not needed anymore and you can add separate license file and remove it from all py files.

**Review by MeetKD — CHANGES_REQUESTED** [review]

**Comment by max3903** [comment]
@OpenCode Please review and merge https://github.com/OpenCode/sale-workflow/pull/2 from @mgosai

**Comment by yelizariev** [comment]
@mgosai you may need to create new PR to OCA as there is no response from @OpenCode

**Comment by mgosai** [comment]
Super seeded by #714

**Comment by tafaRU** [comment]
I close this then. Thank you!

**Pull request closed without merging**